<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_Analysis</class>
 <widget class="QWidget" name="Data_Analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>400</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="plot_reset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ch1_comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>d_x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>theta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>d_theta</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CH1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CH2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ch2_comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>d_x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>theta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>d_theta</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="stop_pushButton">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>360</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
